Update DAML Studio documentation (#6808)
digital-asset/daml: docs/source/daml/daml-studio.rst

================

Top-level declarations of type ``Scenario`` are decorated with a ``Scenario results`` code lens.
You can click on the ``Scenario results`` code lens to inspect the
execution of the scenario.

For the scenario from the :download:`Iou<daml-studio/daml/Iou.daml>`
module, you get the following table displaying all contracts that are
active at the end of the scenario. The first column displays the
contract id. The columns afterwards represent the fields of the
contract and finally you get one column per party with an ``X`` if the
party can see the contract or a ``-`` if not.

.. figure:: daml-studio/images/daml_studio_scenario_table.png
   :scale: 70%
   :align: center

> If you want more details, you can click the *Show archived* and *Show detailed disclosure*
> checkboxes. That will extend the table to include archived contracts as well and
> separate the reason for why a contract is visible into four categories:

Reviewer: Suggest explaining the function of each checkbox separately:

```suggestion
If you want more details, you can click on the *Show archived* checkbox, which extends
the table to include archived contracts, and on the *Show detailed disclosure* checkbox,
which displays why the contract is visible to each party, based on four categories:
```
